tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -699,15 +700,27 @@ describe(__filename, () => {
     expect(newState.addonAddedStatus).toEqual(ADDON_ADDED_STATUS_PENDING);
   });
 
-  it('displays a notification after an add-on has been added', () => {
-    const root = render({});
+  it('displays a notification for 5 seconds after an add-on has been added', () => {
+    const setTimeoutSpy = sinon.spy();
+    const root = render({ setTimeout: setTimeoutSpy });
 
     expect(root.find(Notice)).toHaveLength(0);
 
     root.setProps({ hasAddonBeenAdded: true });
 
     expect(root.find(Notice)).toHaveLength(1);
     expect(root.find(Notice).children()).toHaveText('Added to collection');
+
+    expect(root).toHaveState('addonAddedStatus', ADDON_ADDED_STATUS_SUCCESS);
+    sinon.assert.calledWith(setTimeoutSpy, root.instance().resetMessageStatus, 5000);
+
+    // Simulate the setTimeout behavior.
+    root.instance().resetMessageStatus();
+    // See: https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#for-mount-updates-are-sometimes-required-when-they-werent-before
```

Comment:
Maybe we can use this link with a git tag in the url: https://github.com/airbnb/enzyme/blob/enzyme%403.3.0/docs/guides/migration-from-2-to-3.md#for-mount-updates-are-sometimes-required-when-they-werent-before. That way it should always be valid.